practice_projects/#Homework_pt.py

```diff
@@ -0,0 +1,34 @@
+#Homework pt.1
+
+Friend= "\nSoftware is my friend!\n"
+print (Friend)
+
+#Homework pt.2
+
+Soft = "Software is"
+my = "my friend\n"
+print (Soft,my)
```

Comment:
This prints as:
"Software ismy friend"

How can we fix this?

Reply:
For whatever reason it doesn't print that way on my terminal. I added a space so that shouldn't happen anymore.